server/Dockerfile

```diff
@@ -47,7 +47,7 @@ COPY runtime /
 COPY target /usr/local/
 
 ENV MS_DEBUGLEVEL=0 \
-    MS_ERRORFILE=/var/log/mapserver.log \
+    MS_ERRORFILE=stderr \
```

Comment:
Then you should remove the hack that creates and pipes this file to stdout.

Reply:
Done :-)